ansible/module_utils/debug_utils.py

```diff
@@ -0,0 +1,15 @@
+import os
+from pprint import pprint
+from ansible.module_utils.basic import datetime
+
+def create_debug_file(debug_fname):
+    curtime = datetime.datetime.now().isoformat()
+    timed_debug_fname = debug_fname % curtime
```

Comment:
This requires that the `debug_fname` must be a template string. Without looking into the code, user may pass in an ordinary string as debug file name. Can this be improved?

Reply:
Added add_timestamp option to create_debug_file which defaults to True. If specified it will add the timestamp in between the extension (if present) and the base file name. Otherwise it will append timestamp to the end of the passed filename. 

If add_timestamp is False, we will create the file with the passed filename itself with no timestamp.